CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+# v5.0.0-beta.7
+
+## @rjsf/core
```

Comment:
Technically the next release is v5.0.0-beta.6, so if you move this down around line 26 that would be awesome!